tests/stress/km/stress_tests_km.cpp

```diff
@@ -1048,6 +1050,283 @@ _print_test_control_info(const test_control_info& test_control_info)
     }
 }
 
+static void
+_cleanup_tailcall_program(const fd_t& prog_map_fd)
+{
+    // Cleanup tail calls
+    for (int index = 0; index < MAX_TAIL_CALL_PROGS; index++) {
+        REQUIRE(bpf_map_update_elem(prog_map_fd, &index, &ebpf_fd_invalid, 0) == 0);
+    }
+
+    LOG_VERBOSE("{} Cleanup done for map={}", __func__, prog_map_fd);
+}
+
+static fd_t
+_setup_tailcall_program(bpf_object* object, const std::string map_name)
+{
+    REQUIRE(object != nullptr);
+
+    fd_t prog_map_fd = bpf_object__find_map_fd_by_name(object, map_name.c_str());
+    if (prog_map_fd < 0) {
+        LOG_ERROR(
+            "({}) FATAL ERROR: bpf_object__find_map_fd_by_name({}) failed. Errno:{}",
+            __func__,
+            map_name.c_str(),
+            errno);
+        REQUIRE(prog_map_fd >= 0);
+    }
+    LOG_VERBOSE("({}) Opened fd:{} for map:{}", __func__, prog_map_fd, map_name.c_str());
+
+    // Setup tail calls
+    for (int index = 0; index < MAX_TAIL_CALL_PROGS; index++) {
+        try {
+            std::string bind_program_name{"BindMonitor_Callee"};
+            bind_program_name += std::to_string(index);
+
+            // Try to get a handle to the 'BindMonitor_Calleexxindex' program.
+            bpf_program* callee = bpf_object__find_program_by_name(object, bind_program_name.c_str());
+
+            if (callee == nullptr) {
+                LOG_VERBOSE("({}) - bpf_object__find_program_by_name() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee != nullptr);
+            }
+            LOG_VERBOSE("({}) - bpf_object__find_program_by_name() success.", bind_program_name);
+
+            fd_t callee_fd = bpf_program__fd(callee);
+            if (callee_fd < 0) {
+                LOG_VERBOSE("({}) - bpf_program__fd() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee_fd > 0);
+            }
+            LOG_VERBOSE("({}) - bpf_program__fd() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+            uint32_t result = bpf_map_update_elem(prog_map_fd, &index, &callee_fd, 0);
+            if (result < 0) {
+                LOG_VERBOSE("({}) - bpf_map_update_elem() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(result == ERROR_SUCCESS);
+            }
+            LOG_VERBOSE("({}) - bpf_map_update_elem() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+        } catch (...) {
+            // No need to terminate. We don't care about user mode issues here.
+        }
+    }
+
+    return prog_map_fd;
+}
+
+static void
+_invoke_mt_bindmonitor_tailcall_thread_function(thread_context& context)
+{
+    // Test bind
```

Comment:
```suggestion
    // Test bind.
```
(just being consistent with other comments throughout this file which use punctuation)